packages/client/src/components/forum-thread/forum-thread.module.pcss

```
@@ -0,0 +1,35 @@
+.container {
+  padding-top: 24px;
+}
+
+.panel {
+  padding: 24px;
+  max-width: 930px;
+  margin: 0 auto;
+  border-radius: 12px;
+  background-color: var(--color-light-primary);
+  height: calc(100vh - 2 * 24px);
+}
+
+.link {
+  color: var(--color-accent-primary);
+  padding-bottom: 24px;
+}
```

У тебя иконка стрелки не выровнена относительно текста в ссылке
 ```suggestion
.link {
  color: var(--color-accent-primary);
  padding-bottom: 24px;
  display: inline-flex;
  align-items: center;
}
```